code/modules/vending/vending.dm

```diff
@@ -1622,13 +1633,17 @@
 					icon_state = "pizza-vend"
 					src.generate_HTML(1)
 					updateUsrDialog()
-					sleep(20 SECONDS)
+					if (!src.sharpen)
```

Comment:
why

Reply:
If it's not sped up, the pizza vending machine would spit out a pizza at someone's position 20 seconds later.

Reply:
but why only for the not sharpened case?

Reply:
The 20 second delay is for the cooking for regular pizza vending machines, and [as of the most recent commit] after the vending for sharpened pizza vending machines.